RetailCoder.VBE/App.cs

```diff
@@ -44,171 +34,27 @@ public sealed class App : IDisposable
 
         public App(IVBE vbe, 
             IMessageBox messageBox,
-            IParseCoordinator parser,
             IGeneralConfigService configService,
             IAppMenu appMenus,
             RubberduckCommandBar stateBar,
             IRubberduckHooks hooks,
             IVersionCheck version,
```

Comment:
I need to turn the version check into a *menu* command, and remove the `IVersionCheck` and `CommandBase` parameters - then we're down to 5 and that would be my upper limit for the number of dependencies a cohesive class might need to have.